facade/service.go

```diff
@@ -1228,7 +1231,11 @@ func (f *Facade) scheduleService(ctx datastore.Context, tenantID, serviceID stri
 
 	// Build a list of services to be scheduled
 	svcs := []*service.Service{}
+	var svcIDs []string
 	visitor := func(svc *service.Service) error {
+		if svc.ID != serviceID && svc.Launch == commons.MANUAL && !emergency {
```

Comment:
If it is an emergency shutdown, we want to shut it down even if Launch is set to MANUAL.